projects/plugins/jetpack/extensions/blocks/payments-intro/edit.js

```diff
@@ -34,14 +40,33 @@ export default function JetpackPaymentsIntroEdit( { name, clientId, className }
 		}
 	} );
 
-	const renderVariationPicker = () => {
+	const displayVariations = usableVariations.length && registerBlockVariation;
+
+	let instructions = __( "Please select which kind of payment you'd like to add", 'jetpack' );
+	if ( hasPatterns ) {
+		instructions = __( 'Start by choosing one of our suggested layout patterns', 'jetpack' );
+	}
```

Comment:
If hasPatterns is true we will perform two translations but we will only show the last one.

Reply:
I think this is needed to avoid a code minification issue with the translations. See p1651160960858009/1651157466.974179-slack-C02V49GM0PN

Reply:
Weirdly I had no error when building locally, but as Miguel pointed out, this has been picked up as that error in the build. I've gone back to the original code.